src/components/stepforms/widgets/Replace.vue

```diff
@@ -42,13 +41,28 @@ export default class ReplaceWidget extends Vue {
   @Prop({ type: Array, default: () => [] })
   errors!: ErrorObject[];
 
-  toReplace: any[] = [...this.value];
+  created() {
+    this.update(this.value);
```

Comment:
Only when the value is the default one?
Otherwise it's unnecessary I think
```suggestion
  if(isEqual(this.value, ['', ''])) {
    this.update(this.value);
  }
```